ihmc-perception/src/main/java/us/ihmc/perception/neural/FootstepPredictor.java

```diff
@@ -33,131 +33,133 @@ public class FootstepPredictor
    private static final int IMAGE_INPUT_HEIGHT = 201;
    private static final int IMAGE_INPUT_WIDTH = 201;
 
+   // TODO: We should use pytorch/opencv instead of ONNX. ONNX is a huge dependency we don't use anywhere else and we already have opencv and heavily use that.
```

Comment:
I think you should be clear that ONNX models can still be used, just fed into OpenCV instead of the Microsoft runtime.